spk/jellyfin/PLIST

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Please discard empty line so that file has 0-byte size, like others when empty in spksrc.